tests/data/invalid/transaction/invalid_id.json

```diff
@@ -4,5 +4,6 @@
     "type": "request",
     "duration": 1.0,
     "timestamp": "2017-05-30T18:53:27.154Z",
-    "spans": []
+    "spans": [],
+    "sampled": true
```

Comment:
please remove as it is not a required field, also from the other test json files.